testing/tests/ui/filter_block_ws.stderr

```diff
@@ -1,5 +1,9 @@
-error: failed to parse template source at row 1, column 27 near:
-       " %}\nHELLO\n{{v}}\n{%- endfilter %}"
+error: failed to parse template source
+ --> <source attribute>:1:28
```

Comment:
For this one we could actually get the current file name.

Reply:
I don't think that we from which file we were called, do we? Not until <https://github.com/rust-lang/rust/issues/54725> lands. And the path is written just below this error message, so I don't think that is matters much.